### PR TITLE
nautilus: mon scrub testing

### PR DESCRIPTION
**Pull request by badone**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: qa/suites/rados/monthrash/ceph.yaml [hunk 1]
```diff
@@ -8,8 +8,12 @@ overrides:
         mon osdmap full prune min: 15
         mon osdmap full prune interval: 2
         mon osdmap full prune txsize: 2
+        mon scrub inject crc mismatch: 0.01
+        mon scrub inject missing keys: 0.05
 # thrashing monitors may make mgr have trouble w/ its keepalive
-    log-whitelist:
+    log-ignorelist:
```
Comment (jdurgin) [inline qa/suites/rados/monthrash/ceph.yaml:14]: nautilus doesn't have ignore-list

Comment (badone) [reply qa/suites/rados/monthrash/ceph.yaml]: Damn, reversed the conflict. Thanks @jdurgin

**Comment by badone** [comment]
jenkins test make check

**Comment by badone** [comment]
jenkins test api

**Review by jdurgin — APPROVED** [review]

**Comment by badone** [comment]
jenkins test docs

**Pull request merged**